.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: php
-
-sudo: false
+os: linux
```

Comment:
@DamienHarper What do you think about use the `dist: xenial` ?

Reply:
Xenial is currently [Travis CI's default linux distro](https://docs.travis-ci.com/user/reference/overview/#linux), not sure we have to explicitly lock to this one (apart from not risking to get failed builds the day Xenial is not Travis's default linux distro)

Reply:
yep agree, u can see it [here](https://docs.travis-ci.com/user/reference/overview/#for-a-particular-travisyml-configuration)
Also this [link](https://docs.travis-ci.com/user/reference/overview/#deprecated-virtualization-environments) you can find the reference to deprecated ``sudo`` key